### DISCUSSION TITLE
Installing TabPy on a shared host

### DISCUSSION THREAD

**Discussion by kwculp** [tableau/TabPy#563] [Q&A] [answered]
Hello,

We’re exploring the capabilities of TabPy integration with Tableau Desktop and would like to install TabPy on a shared host (such as Bluehost.com).

Can this be done? Does anyone have instructions on doing this? I would need to use the host name to access it and not the (shared) numeric IP address.

Thank you,
Kevin

**Comment by kwculp** [comment, marked as answer]
Well, given the lack of responses, this must not be a common thing to do.

I managed to get it to work, but only by installing Ubuntu on my Raspberry Pi. Using the Raspberry Pi OS gave me too many install errors but Ubuntu worked flawlessly.

**Comment by 0golovatyi** [comment]
@kwculp  You can use hostname instead of IP address, but that name has to be for a host and not include a path. E.g. myserver.mydomain.com will work and myserver.mydomain.com/myapp/myversion won't. http://tabscifi.golovatyi.info/2020/03/tableau-desktop-2020-1-advanced-analytics-improvements/.
